scipy/stats/_discrete_distns.py

```diff
@@ -514,6 +514,121 @@ def _logcdf(self, k, M, n, N):
 hypergeom = hypergeom_gen(name='hypergeom')
 
 
+class nhypergeom_gen(rv_discrete):
+    r"""A negative hypergeometric discrete random variable.
+
+    The negative hypergeometric distribution models the probability
```

Comment:
I do not understand the definition. Wikipedia: "There are N elements, of which K are defined as "successes" and the rest are "failures". Elements are drawn one after the other, without replacements, until r failures are encountered. Then, the drawing stops and the number k of successes is counted. The negative hypergeometric distribution is the distribution of this k"

Reply:
I think it just means that we choose one element from the box without replacement until we end up with `r` elements which we consider failures (like getting `r` blue balls while targeting for red balls) and then we count how many successes `k` we encountered (like getting `k` red balls (successes) in a sample with `r` blue balls (failures)). In the end, there are `k+r` samples collected without replacement and `nhypergeom` is the distribution over this `k`. I hope this clears that out. I will add this in the tutorial.